### PR TITLE
Now we print a stack trace on copyright errors

### PR DESCRIPTION
**Pull request by digeff**
We can use this to debug errors on our build system

### CODE REVIEW

File: tools/checkCopyright.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ var CopyrightVerifier = (function () {
             return defer.resolve(foundFiles);
         });
         finder.on("patherror", function (err, strPath) {
-            defer.reject("Error for Path " + strPath + " " + err);
+            defer.reject("Error for Path " + strPath + " " + err + "\nStackTrace: " + err.stack);
```
Comment (dlebu) [inline tools/checkCopyright.js]: Check err for `null`?

**Comment by dlebu** [comment]
Minor comment, LGTM.

File: tools/checkCopyright.js (hunk 1 above)
Comment (digeff) [inline tools/checkCopyright.js]: The docs don't really say, but I'm choosing to believe that err will never be null. Why would you call a patherror event if you don't have an error?

**Comment by MSLaguana** [comment]
looks good to me

**Pull request merged**